pkg/testutil/env.go

```diff
@@ -23,8 +23,6 @@ type EnvOpts struct {
 
 // NewEnv creates a new test environment and returns the root directory.
 func NewEnv(opts EnvOpts) (rootdir string) {
-	opts.T.Helper()
```

Comment:
A `Helper` marks the calling function as a test helper function so, when printing file and line information, that function will be skipped. I decided I wanted to see that output.